### PR TITLE
Added LICENSE

### PR DESCRIPTION
**Pull request by rmeissner**
Changes proposed in this pull request:
- Added Apache 2 as a licence

@gnosis/mobile-devs

### CODE REVIEW

**Comment by ligi** [comment]
just wondering: is there a reason you choose apache over MIT? I use MIT in kethereum - wonder why you where choosing apache for this project. I think it shoud be compatible - but IANAL and do not really want to risk things or even spend cognitive resources on this. If there are good reasons for apache I might consider  a switch as long as it is still easy as the contributors-list to ask is still not that large ..-)

**Comment by rmeissner** [comment]
We choose apache because it is pretty common for Android projects (at least the ones we use ;) ). As the whole licence situation is still ongoing (since we want to have some common ground for all gnosis repos) the licence might actually change to MIT. If we find out why you should either one over the other one, I will let you know :D

TL;DR: No good reason why to choose Apache over MIT ;)
